src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/DataApiCommandSenders.java

```diff
@@ -0,0 +1,12 @@
+package io.stargate.sgv2.jsonapi.api.v1.util;
+
+/** Helper class used for constructing and sending commands to the Data API */
+public class DataApiCommandSenders {
+  public static DataApiNamespaceCommandSender namespaceCommand(String namespace) {
+    return new DataApiNamespaceCommandSender(namespace);
+  }
+
+  public static DataApiTableCommandSender tableCommand(String namespace, String tableName) {
```

Comment:
idea - if we name these assertTableCommand, then do static imports in tests, and make the other name change below it could look like this in a test: 

```
assertTableCommand(namespace, table)
    .findOne({})
```

Reply:
Great idea, will change method name.

Could change class name too, to `DataApiCommandAssertions` too?

Reply:
The only downside is that these look odd as set up components (creating table, inserting rows, that are not part of test).
Will change, we can re-think if need be.